.github/workflows/indent.yml

```diff
@@ -0,0 +1,24 @@
+name: Indent
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ master, develop ]
```

Comment:
For some reason I named the default branch now `main` and not `master`

Reply:
New default in git. The other action should run though.

Reply:
Wait, I guess I need to enable the GH CI explicitly.

Reply:
..maybe you need more permission to do so. I cannot trigger the CI here manually. Just invited you as collaborator, which might help.